### ISSUE TITLE
BART 발표를 위한 링크 모음

### ISSUE THREAD

**Issue by jinmang2** [jinmang2/bring_it_on#4]
https://github.com/pytorch/fairseq/blob/aa79bb9c37b27e3f84e7a4e182175d3b50a79041/fairseq/data/denoising_dataset.py

https://github.com/pytorch/fairseq/blob/aa79bb9c37b27e3f84e7a4e182175d3b50a79041/fairseq/data/encoders/utils.py

https://github.com/pytorch/fairseq/blob/aa79bb9c37b27e3f84e7a4e182175d3b50a79041/fairseq/tasks/denoising.py#L27

https://github.com/huggingface/transformers/issues/5428

https://github.com/huggingface/transformers/issues/1676

https://github.com/huggingface/transformers/pull/2745

https://github.com/pytorch/fairseq/issues/1762

https://github.com/pytorch/fairseq/blob/master/examples/bart/README.md

https://github.com/modulabs/beyondBERT/issues/6

https://towardsdatascience.com/bart-for-paraphrasing-with-simple-transformers-7c9ea3dfdd8c

https://github.com/huggingface/transformers/issues/5428

https://datascience.stackexchange.com/questions/9302/the-cross-entropy-error-function-in-neural-networks

https://towardsdatascience.com/blender-bot-part-1-the-data-524beaedde65

http://people.csail.mit.edu/tianxiao/data/

https://www.aclweb.org/anthology/2020.acl-main.703.pdf

https://github.com/shentianxiao/text-autoencoders/blob/master/train.py

https://github.com/shentianxiao/text-autoencoders/blob/master/model.py

https://jeongukjae.github.io/posts/layer-normalization/

https://blog.airlab.re.kr/2019/08/Group-Normalization

https://stackoverflow.com/questions/59830168/layer-normalization-in-pytorch

https://dladustn95.github.io/nlp/BART_paper_review/

https://discuss.pytorch.org/t/crossentropy-loss-for-rnn-output/3029

https://github.com/pranjaldatta/Denoising-Autoencoder-in-Pytorch/blob/master/DenoisingAutoencoder.ipynb

https://sshleifer.github.io/blog_v2/jupyter/2020/03/12/bart.html
